### ISSUE TITLE
Error libtool

### ISSUE THREAD

**Issue by ludwigschreier** [emul8/emul8#136]
Hi,
I'm trying to install on Ubuntu 16.10 64bit, but this happens:

```
~/github/mono$ ./autogen.sh --prefix=/usr
**Error**: You must have `libtool' installed to compile Mono.
Get ftp://ftp.gnu.org/pub/gnu/libtool-1.2d.tar.gz
(or a newer version if it is available)
```

```
ludwigschreier@ubuntu-desktop1610:~/github/mono$ sudo apt install libtool
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libtool is already the newest version (2.4.6-1).
0 upgraded, 0 newly installed, 0 to remove and 23 not upgraded.
```

BR

**Comment by PiotrZierhoffer** [comment]
Hi

Thanks for your report.

This looks strange, as a workaround I'd suggest installing mono from [PPA shipped by Xamarin](http://www.mono-project.com/docs/getting-started/install/linux/#debian-ubuntu-and-derivatives).

We're currently working on excluding the "Mono from sources" part from the building process.

Please let us know if everything works on the precompiled Mono.

Best regards

**Comment by rasomc** [comment]
Hey folks,

Had the same issue on Ubuntu 16.04. I adapted the autogen.sh since the which command in line 36 fails.

`if [ ! -x "$LIBTOOL" ]; then
    LIBTOOL=`which libtool`
  fi`

I replaced it with:
`if [ ! -x "$LIBTOOL" ]; then
    LIBTOOL=`which libtoolize`
  fi`

and removed the concatenation on line 101:

`${LIBTOOL}ize --force --copy` replaced with `${LIBTOOL} --force --copy`

It then works but I was not able to build the required version of mono.

I was able to build emul8 with the mono version in the Ubuntu repo (Version 2.4.6-0.1) installed via:
`sudo apt-get libtool` exactly the same way you did it.

Had an issue later on with emul8 but in general it seems to work without the specifiv mono verision. At least the samples worked fine.
